website/index.qmd

```diff
@@ -21,6 +15,10 @@ The CRAN Cookbook has been developed as part of a special project grant from the
 
 # Prerequisites
 
+1. **Beginner to intermediate knowledge of R Programming**: This guide is designed to assist both first-time R package maintainers and those with experience who prefer to navigate a curated selection of the most common issues encountered when submitting to CRAN, as identified by CRAN Maintainers. It provides a 'best practices' approach to resolving R package issues to facilitate a successful re-submission to CRAN.
+
+2. **Experience in debugging code**: Submitting your R package to CRAN is a multi-step process that may involve communicating with the assigned CRAN volunteer and updating your code, tests, examples, documentation, and other supplementary files to meet submission criteria. Developing strong debugging skills can be particularly valuable during this process, as it can help you effectively apply the examples in this cookbook to create custom solutions for updating your R package.
```

Comment:
Nice addition about the prerequisites!

1) Sometimes the CRAN volunteer changes when discussing the same issue. I've recently seen messages between a maintainer and CRAN about an issue that were initiated by a CRAN volunteer and were later follow up by another (It was not a submission of a new version of a package). Is there an assigned "CRAN volunteer" for all issues or only for submissions?
2) In the first line you mention that it might involve "multi-step process ... to meet submission criteria", shouldn't that be **acceptance** criteria?

Reply:
1.: Sorry I mixed things: In the introduction it is mentioned that the goal is to "Educate a diverse cohort of young R developers to acquire the technical knowledge and skills required to participate in the continuous integration of R packages into the R ecosystem." but in the first paragraph of the prerequisites it mentions "most common issues encountered when submitting to CRAN, as identified by CRAN Maintainers.".

If a maintainer hasn't had issues submitting or getting the package accepted they can still need some skills to understand the "continuous integration of R packages" done by CRAN. Because a lot of people have problems being accepted to CRAN, but [>80% of them make it through the review](https://user2021.llrs.dev/#14) and the biggest problem is that [60% of those archived](https://www.cranhaven.org/cran-archiving-stats.html#packages-archived-over-all) and never going back to CRAN. Many are due to maintainers not being responsive or other external issues but mostly because it is hard to avoid being archived and how to fix issues that led to archival (and those issues come from the cran@r-project.org email). For example, [more than 200 packages were](https://www.cranhaven.org/cran-archiving-stats.html#age-of-archival) archived in less than two weeks after being accepted on CRAN, because CRAN Maintainers found some issues on them (Of them ~100 are package on their first submission). Kurt Hornik, in a recent exchange, mentioned that this is because on the submission not everything can be check.

Maybe editing to mention that it might also help with preventing being archived would help (aka, also implying that one could also have a package on CRAN to find the guide useful).  

@jasdumas Thanks for the acknowledgements!